src/Util/Finder.php

```diff
@@ -4,8 +4,22 @@
 
 use Exo\History;
 
-class Finder
-{
+class Finder {
```

Comment:
The opening brace should be on the next line for code-style consistency.

Reply:
Right on, this is usually auto-handled for me, so I guess Ive gotten lazy. I would've thought code lint would catch that, but it may not be enabled on this repo yet?

Reply:
Done, see latest commit.